publication-commons/src/main/java/no/unit/nva/publication/model/business/PublishingRequestCase.java

```diff
@@ -149,19 +138,10 @@ public void validateCompletionRequirements(Publication publication) {
     @Override
     public PublishingRequestCase complete(Publication publication, Username finalizedBy) {
         var completed = (PublishingRequestCase) super.complete(publication, finalizedBy);
-        completed.setApprovedFiles(getFilesToApprove(publication));
```

Comment:
We cant do it by default longer, cause we have to manage which files to approve when updating publication. Cause different institutions are now responsible for files uploaded by institution users (curators/contributors)